pages/resources/quests.mdx

```diff
@@ -0,0 +1,329 @@
+
+###### Example Quest Reward Code
+
+```json
+{
+  "quest_id": "8206816794116096000",
+  "code": "111-1111111",
+  "platform": 0,
+  "user_id": "222069018507345921",
+  "claimed_at": "2077-01-01T18:41:29.706194+00:00",
+  "tier": null
+}
+```
+
+### Quest Claimed Reward Object
+
+Represents the claimed rewards.
+
+##### Quest Claimed Reward Structure
+
+| Field        | Type                                                       | Description                                                    |
+| ------------ | ---------------------------------------------------------- | -------------------------------------------------------------- |
+| claimed_at   | ISO8601 timestamp                                          | When the reward was claimed                                    |
+| entitlements | array[entitlement object]                                  | The entitlements the user received                             |
+| errors       | array[[JSON error](/topics/opcodes-and-status-codes#json)] | An array of errors that have occured while claiming the reward |
+
+##### Example Quest Claimed Reward
+
+```json
+{
+  "claimed_at": "2024-04-17T23:30:41.000321+00:00",
+  "entitlements": [
+    {
+      "id": "1230299425620885624",
+      "sku_id": "1226939756617793606",
+      "application_id": "1242265603276800000",
+      "user_id": "222069018507345921",
+      "deleted": false,
+      "type": 10,
+      "tenant_metadata": {},
+      "gift_code_flags": 0,
+      "promotion_id": null
+    }
+  ],
+  "errors": []
+}
+```
+
+## Endpoints
+
+<RouteHeader method="GET" url="/quests/@me">
+  Get Available Quests
+</RouteHeader>
+
+Returns a list of all available [quests](#quest-object).
+
+<RouteHeader method="GET" url="/assets/quests/{quest.id}/{theme}/game_tile.png">
+  Get Quest Asset
+</RouteHeader>
+
+Returns the specified image asset for the theme (e.g dark/light)
+
+<RouteHeader method="POST" url="/quests/{quest.id}/enroll">
+  Accept Quest
+</RouteHeader>
+
+Accepts a quest and returns a [quest user status](#quest-user-status-object) object.
+
+<RouteHeader method="POST" url="/quests/{quest.id}/claim-reward">
+  Claim Quest Reward
+</RouteHeader>
+
+Claims the quest's rewards, setting the `completed_at` and `claimed_at` fields of the [user status](#quest-user-status-object) to the current timestamp, and returns the [claimed rewards](#quest-claimed-reward-object).
+
+<RouteHeader method="POST" url="/quests/{quest.id}/reward-code">
+  Get Quest Reward Code
+</RouteHeader>
+
+Retrieves the reward code for the specified platform and returns a [quest reward code](#quest-reward-code-object) object.
+
+###### JSON Params
+
+| Field    | Type    | Description                                                   |
+| -------- | ------- | ------------------------------------------------------------- |
+| location | integer | Used for analytics (probably)                                 |
```

Comment:
Haven't really documented any ?location params anywhere (there's a few scattered around the api) but also not really sure...